components/apimgt/org.wso2.carbon.apimgt.core/pom.xml

```diff
@@ -578,6 +578,9 @@
                                             <ACCEPT_EULA>Y</ACCEPT_EULA>
                                             <SA_PASSWORD>wso2apim123#</SA_PASSWORD>
                                         </env>
+                                        <ports>
+                                            <port>11433:1433</port>
+                                        </ports>
```

Comment:
these values we can add dynamically,
If we add we can't gurantee the port range.